infrastructure/src/main/java/com/example/opencommerce/infra/catalog/ItemSorter.java

```diff
@@ -0,0 +1,30 @@
+package com.example.opencommerce.infra.catalog;
+
+import com.example.opencommerce.app.catalog.query.SearchRequest;
+import com.example.opencommerce.infra.catalog.views.BaseItemView;
+
+import java.util.Comparator;
+import java.util.List;
+
+class ItemSorter {
```

Comment:
Any reason for this t be a bean? It has only one method that could be as well static. It does return a new object, it doesn't change the state of incoming objects, it doesn't use share state. What's the difference between singleton managed by framework, and the static method?